src/components/Dashboard/FilingHistoryList/StaffNotation.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+    <div id="staff-notation" :class="{ 'add-scrollbar-offset': addScrollbarOffset }">
+        <add-staff-notation-dialog
+            :dialog="isAddingRegistrarsNotation"
+            @close="hideRegistrarsNotationDialog($event)"
+            attach="#staff-notation"
+            itemName="Registrar's Notation"
+            filingType="registrarsNotation"
+        />
+        <add-staff-notation-dialog
+            :dialog="isAddingRegistrarsOrder"
+            @close="hideRegistrarsOrderDialog($event)"
+            attach="#staff-notation"
+            itemName="Registrar's Order"
+            filingType="registrarsOrder"
+        />
+        <add-staff-notation-dialog
+            :dialog="isAddingCourtOrder"
+            @close="hideCourtOrderDialog($event)"
+            attach="#staff-notation"
+            itemName="Court Order"
+            filingType="courtOrder"
+            courtOrderNumberRequired="true"
+        />
+        <div class="filing-item__actions">
+            <v-menu offset-y left transition="slide-y-transition">
+                <template v-slot:activator="{ on }">
+                    <span>
+                        <span
```

Comment:
Code smell: why do you have 2 spans? Maybe the outer one can be merged with your template element. Or maybe you can assign a style to change the block-level elements to inline ones. Or both.

Reply:
Yes, I forgot to remove that. 👍